src/utils/timeUtils.ts

```diff
@@ -11,8 +11,8 @@ import { DateTime, Duration } from 'luxon';
 export function floorNearestInterval(dt: DateTime, interval: number): number {
   const unixEpochSeconds = dt.toUnixInteger();
 
-  // we can't do anything with intervals
-  if (interval < 1 || !Number.isSafeInteger(interval)) {
```

Comment:
@eshedmargalit I figured we save some computer when interval is 1

Reply:
Nice!